contentcuration/contentcuration/frontend/shared/mixins.js

```diff
@@ -191,6 +191,468 @@ export const constantsTranslationMixin = {
   },
 };
 
+// METADA STRINGS AND RELATED MIXIN
```

Comment:
Minor, but is this meant to say "METADATA"?

Reply:
It sure is :D I can update